client-cert caching interactions (#1892)
httpwg/http-extensions: draft-ietf-httpbis-client-cert-field.md

`Client-Cert` or `Client-Cert-Chain` header field. Similarly, clients MUST NOT
employ the `Client-Cert` or `Client-Cert-Chain` header field in requests.

> When the value of the `Client-Cert` request header is used to select a response

Reviewer: ```suggestion
When the value of the `Client-Cert` request header field is used to select a response
```

(e.g., the response content is access-controlled), the response MUST either be

> marked as uncacheable (e.g., `Cache-Control: no-store`) or be designated for

Reviewer: It doesn't necessarily need to be _marked_  -- some responses just are uncacheable.

```suggestion
uncacheable (e.g., by sending `Cache-Control: no-store`) or be designated for
```

> selective reuse only for subsequent requests with the same `Client-Cert`
> header value (e.g., `Vary: client-cert`).

Reviewer: `Vary` isn't just an example, it's the mechanism.

```suggestion
selective reuse only for subsequent requests with the same `Client-Cert`
header value by sending a `Vary: Client-Cert` response header.
```
